odoo/addons/base/security/ir.model.access.csv

```diff
@@ -2,6 +2,8 @@
 "access_decimal_precision_config","decimal.precision configuration","model_decimal_precision","group_system",1,1,1,1
 "access_ir_attachment_group_user","ir_attachment group_user","model_ir_attachment","group_user",1,1,1,1
 "access_ir_attachment_group_portal_public","ir_attachment group_portal_public","model_ir_attachment",,0,0,0,0
+"access_ir_async_group_async_user","ir_async group_user","model_ir_async","group_user",1,0,0,0
```

Comment:
I don't think regular users should be able to access `ir.async` at all.  (The `ir.rule` is useless, too.)
This is a purely technical artefact.

Reply:
Actually this is needed because we list background jobs on the frontend in the systray